libraries/protocol/include/golos/protocol/steem_virtual_operations.hpp

```diff
@@ -248,6 +248,21 @@ namespace golos { namespace protocol {
             asset curator_reward;
             int64_t net_rshares;
         };
+
+        // Used only in operation_dump-plugin
+        struct vote_rshares_operation : public virtual_operation {
+            vote_rshares_operation() {
+            }
+
+            vote_rshares_operation(const account_name_type& v, const account_name_type& a, const string& p, int64_t r)
+                    : voter(v), author(a), permlink(p), rshares(r) {
+            }
+
+            account_name_type voter;
+            account_name_type author;
+            string permlink;
```

Comment:
weight is needed too.

Reply:
@afalaleev For what it need in virtual operation? We have it in `vote_operation`.